src/core/core.datasetController.js

```diff
@@ -234,6 +235,47 @@ helpers.extend(DatasetController.prototype, {
 		me.resyncElements();
 	},
 
+	/**
+	 * Private functionality to be called before every update
+	 * @private
+	 */
+	_update: function() {
```

Comment:
We don't need this method anymore per our last Slask discussion (no more dataset merge)

Reply:
Done